internal/controller/employee_test.go

```diff
@@ -1,3 +1,6 @@
+//go:build !integration
+// +build !integration
```

Comment:
These are not integration tests. Pay more attention to the test types. You need specific environment for integration tests (e.g. running database). For unit tests which often utilize mocks you don't need one.